test/e2e/wdio.base.conf.ts

```diff
@@ -0,0 +1,84 @@
+import path from 'path'
+import { unlinkSync, mkdirSync } from 'fs'
+import type { Options, Reporters } from '@wdio/types'
+import { browser, $, $$ } from '@wdio/globals'
+import { getRunId, getTestReportDirectory } from '../envUtils'
+import { APPLICATION_ID } from './lib/helpers/constants'
+
+const reporters: Reporters.ReporterEntry[] = [['spec', { onlyFailures: true }]]
+let logsPath: string
+
+const testReportDirectory = getTestReportDirectory()
+if (testReportDirectory) {
+  reporters.push([
+    'junit',
+    {
+      outputDir: testReportDirectory,
+      outputFileFormat(options) {
+        const browserName = 'browserName' in options.capabilities ? String(options.capabilities.browserName) : 'unknown'
+        return `results-${options.cid}.${browserName}.xml`
+      },
+    },
+  ])
+  logsPath = path.join(testReportDirectory, 'specs.log')
+} else if (!process.env.LOGS_STDOUT) {
+  logsPath = 'specs.log'
+}
+
+export const config: Options.Testrunner = {
+  runner: 'local',
+  autoCompileOpts: {
+    autoCompile: true,
+    tsNodeOpts: {
+      project: './tsconfig.json',
+    },
+  },
+  // We do not inject @wdio globals to keep Jasmine's expect
+  injectGlobals: false,
+  specs: ['./scenario/**/*.scenario.ts'],
+  capabilities: [],
+  maxInstances: 5,
+  logLevel: 'warn',
+  bail: 0,
+  waitforTimeout: 10000,
+  connectionRetryTimeout: 90000,
+  connectionRetryCount: 0,
+  framework: 'jasmine',
+  reporters,
+  jasmineOpts: {
+    defaultTimeoutInterval: 60000,
+  },
+  onPrepare: (_config, _capabilities) => {
+    console.log(
+      `[RUM events] https://app.datadoghq.com/rum/explorer?query=${encodeURIComponent(
+        `@application.id:${APPLICATION_ID} @context.run_id:"${getRunId()}"`
+      )}`
+    )
+    console.log(`[Log events] https://app.datadoghq.com/logs?query=${encodeURIComponent(`@run_id:"${getRunId()}"`)}\n`)
+
+    if (testReportDirectory) {
+      try {
+        mkdirSync(testReportDirectory, { recursive: true })
+      } catch (e) {
+        console.log(`Failed to create the test report directory: ${(e as Error).message}`)
+      }
+    }
+
+    if (logsPath) {
+      try {
+        unlinkSync(logsPath)
+      } catch (e) {
+        if ((e as NodeJS.ErrnoException).code !== 'ENOENT') {
+          console.log(`Failed to remove previous logs: ${(e as Error).message}`)
+        }
+      }
+    }
+  },
+  beforeSession: (_config, _capabilities, _specs, _cid) => {
+    // Export everything besides expect, as we want to keep the one from Jasmine
+    global.browser = browser
+    global.driver = browser
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should we update our `driver` usages to use `browser` instead?